modules/svg/ai_features.js

```diff
@@ -1,6 +1,6 @@
 import _throttle from 'lodash-es/throttle';
 
-import { select as d3_select } from 'd3-selection';
+import { select as d3_select, selectAll as d3_selectAll } from 'd3-selection';
```

Comment:
It doesn't look like `d3_selectAll` is used here anymore, so this can be cleaned up.